src/test/java/com/yahoo/bullet/pubsub/SubscriberTest.java

```diff
@@ -0,0 +1,28 @@
+package com.yahoo.bullet.pubsub;
+
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+
+import java.util.UUID;
+
+public class SubscriberTest {
+    @Test
+    public void testCommitWithNoSequenceNumber() {
+        String randomID = UUID.randomUUID().toString();
+        Subscriber subscriber = Mockito.mock(Subscriber.class, Mockito.CALLS_REAL_METHODS);
```

Comment:
If you test it, you don't have to mock it. Just create a minimal test implementation (can be in this file) and just call the default method if you need it.